src/App.tsx

```diff
@@ -90,11 +91,30 @@ const columns = [{
 {
   dataField: 'name',
   text: 'Product Name',
+  editor: {
+    type: Type.SELECT,
+    options: [{
+      value: 'TV',
+      label: 'TV',
+    }, {
+      value: 'Mobile',
+      label: 'Mobile',
+    }, {
+      value: 'Book',
+      label: 'Book',
+    }],
+  },
 }, {
   dataField: 'price',
   text: 'Product Price',
   sort: true,
 }];
+const cantEdit = new Set([0]);
```

Comment:
if you initialize a set, isn't it already empty?

Reply:
Wanted column of index 0 (first column) to be non-editable!